plugins/org.python.pydev.parser/src/org/python/pydev/parser/visitors/scope/EasyAstIteratorBase.java

```diff
@@ -276,6 +276,9 @@ public void traverse(FunctionDef node) throws Exception {
             node.name.accept(this);
         if (node.args != null)
             node.args.accept(this);
+        if (node.returns != null) {
```

Comment:
Great that the fix was straightforward... still, this should have a test along with it.

I think you can create a `RefactoringRequest` and check `PyReferenceSearcher.getLocalReferences` directly (the hardest thing is probably creating the `RefactoringRequest`, but that should be similar to what you did in the other pull request0.